definitions/divorce/json/CaseType.json

```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "DIVORCE",
-    "Name": "Divorce case - v114.7",
+    "Name": "Divorce case - v114.8",
```

Comment:
Tony's PR already has this version:
https://github.com/hmcts/div-ccd-definitions/pull/94/files#diff-94444d93f934f87c887953da988c8b6fR5

Reply:
Who doesn't merge first will have to change haha. The race is on.